listen.js

```diff
@@ -0,0 +1,8 @@
+const app = require("./app");
```

Comment:
we don't need a listen file yet until we start to think about hosting, supertest has its own built in listener